api/v1/harborcluster_types.go

```diff
@@ -267,6 +271,7 @@ type PostgresSQL struct {
 	Replicas         int                         `json:"replicas,omitempty"`
 	Version          string                      `json:"version,omitempty"`
 	StorageClassName string                      `json:"storageClassName,omitempty"`
+	Image            string                      `json:"image,omitempty"`
```

Comment:
I don't think we should tinker with code just because some people can't pull an image from China, after all, there are many other users. Is it possible to take through the document or use other configuration files to solve this problem?

Reply:
After carefully thinking, maybe keeping the current way is an acceptable solution.